cedar-policy-validator/src/diagnostics/validation_errors.rs

```diff
@@ -72,20 +73,55 @@ pub struct UnrecognizedActionId {
     pub actual_action_id: String,
     /// An action id from the schema that the user might reasonably have
     /// intended to write.
-    pub suggested_action_id: Option<String>,
+    pub unrecognized_action_id_help: Option<UnrecognizedActionIdHelp>,
 }
```

Comment:
Probably worth updating the doc comment.

Reply:
Updated and renamed this field to `hint` as we do in other types.